app/models/tag.rb

```diff
@@ -10,6 +10,11 @@ class Tag < ActsAsTaggableOn::Tag
   include Purgeable
   include Searchable
 
+  include PgSearch::Model
+  pg_search_scope :search_by_name,
```

Comment:
I know you want to measure performance in production, but pretty much every time I built a Postgres FTS I ended up adding `tsvector` columns:

https://github.com/Casecommons/pg_search#using-tsvector-columns

That said, the generated query seems fast enough in Blazer.